projects/orquestra-sdk/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 🚨 *Breaking Changes*
 
+* Installing `orquestra-sdk[ray]` no longer installs Ray dashboard. Use `orquestra-sdk[all]` or `orquestra-sdk[dashboard]` instead.
```

Comment:
Just a note: I forgot to put it here in my previous PR. Adding it as a drive-by fix

Reply:
In the past, we considered making automated changelog based on PR titles, and concluded that it's better to have a separate, hand-crafted file. IIRC, the reasoning was:
* it doesn't make senses for every PR to be part of the release notes
* the context is different. PR titles tend to be tailored for easy PR searching in history, or for onboarding code reviewers. The changelog uses a much different style, and often includes usage snippets
* it was difficult to enforce the rule of "PR title should read good as a changelog entry", especially when handling PRs made from infrequent, external contributors

I believe most of this reasoning also applies to commit messages, so I'd stick with hand-crafted release notes.

Reply:
However! I'd be curious to explore automated version bumps based on commit messages on `main`. We squash branch content when we merge PRs, so this would mean relying on PR titles for inferring appropriate semver bump in the next release.